src/ixth-expansions/lost-star-charts.js

```diff
@@ -0,0 +1,329 @@
+                legendary: true,
+                legendaryCard: "card.ghoti:homebrew.ixth_lost_star_charts/legendary_lloyd_planet",
+            },
+        ],
+    },
+    {
+        tile: 4012,
+        source: "homebrew.ixth_lost_star_charts",
+        packageId: refPackageId,
+        img: "ixth-expansions/lost-star-charts/tiles/tile_4012.jpg",
+        planets: [
+            {
+                localeName: "planet.windmore",
+                resources: 3,
+                influence: 2,
+                trait: ["cultural"],
+                legendary: true,
+                legendaryCard: "card.ghoti:homebrew.ixth_lost_star_charts/legendary_windmore_planet",
+            },
+        ],
+    },
+    {
+        tile: 4013,
+        source: "homebrew.ixth_lost_star_charts",
+        packageId: refPackageId,
+        img: "ixth-expansions/lost-star-charts/tiles/tile_4013.jpg",
+        planets: [
+            {
+                localeName: "planet.ethan",
+                resources: 6,
+                influence: 0,
+                trait: ["hazardous"],
+                legendary: true,
+                legendaryCard: "card.ghoti:homebrew.ixth_lost_star_charts/legendary_ethan_planet",
+            },
+        ],
+        wormholes: ["alpha", "beta"],
+        offMap: true,
+    },
+    {
+        tile: 4014,
+        source: "homebrew.ixth_lost_star_charts",
+        packageId: refPackageId,
+        img: "ixth-expansions/lost-star-charts/tiles/tile_4014.jpg",
+        planets: [
+            {
+                localeName: "planet.alpert",
+                resources: 3,
+                influence: 0,
+                trait: ["cultural"],
+                legendary: true,
+                legendaryCard: "card.ghoti:homebrew.ixth_lost_star_charts/legendary_alpert_planet",
+            },
+        ],
+    },
+    {
+        tile: 4015,
+        source: "homebrew.ixth_lost_star_charts",
+        packageId: refPackageId,
+        img: "ixth-expansions/lost-star-charts/tiles/tile_4015.jpg",
+        anomalies: ["asteroid field", "gravity rift"],
+    },
+    {
+        tile: 4016,
+        source: "homebrew.ixth_lost_star_charts",
+        packageId: refPackageId,
+        img: "ixth-expansions/lost-star-charts/tiles/tile_4016.jpg",
+        wormholes: ["alpha", "beta"],
+        offMap: true,
+    },
+];
+
+const attachments = [
+    {
+        packageId: refPackageId,
+        localeName: "attachment:homebrew.ixth_lost_star_charts.terraforming_initiatives/cultural_boom",
+        cardNsid: "card.terraforming_initiatives:homebrew.discordant_stars/cultural_boom", // FIXME: add card
+        tokenNsid:
+            "token.attachment:homebrew.ixth_lost_star_charts.terraforming_initiatives/cultural_boom", // FIXME: add token
+        faceUp: {
+            influence: 2,
+            traits: ["cultural"],
+            image: "discordant-stars/extras/bentor_encryption_key.png",
+        },
+    },
+];
+
+// TODO: explain how to add exploration cards
```

Comment:
you can call world.TI4.homebrew.resetOnTableDecks() to regenerate the exploration decks (and all decks), adding in new any new cards you registered.

Reply:
Done